query-tuning/index.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Query Tuning'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+import DocsBetaFeature from '../components/DocsBetaFeature'
+
+<DocsBetaFeature feature="Query Tuning" />
+
+## Getting Started
+
+Query Tuning provides a unified interface and workflow for optimizing slow Postgres queries.
+
+The Query Tuning Workbooks feature lets you streamline recording of related `EXPLAIN ANALYZE`
+outputs and enables easy, secure sharing them with your team. Each workbook contains a
+baseline query and one or more variants.
+
+Variants are a modification to the original query that aims to optimize it by changing planner
+settings, adding a `pg_hint_plan` hint, or rewriting the query. You can compare and analyze
+variants with clear visibility into changes and their effects on performance.
+
+The parameter sets feature makes it easy to test performance for different bind parameter values
+across the baseline and variants, making sure that optimizing for one user's experience does not
+slow down other cases.
+
+Query Tuning workbooks automatically convert positional parameters to named parameters, e.g. turning
+`user_id = $1` into `user_id = $user_id`. This makes it easy to test rewrites of a query, or test
+individual portions of a query.
+
+### Query Tuning Features
```

Comment:
This feels a bit more like marketing than like documentation. I can see why we want to list it, but it kind of seems out of place here...

Reply:
Yeah, this was from the original docs draft and I also had that feedback then - let me see if I can cut it down / rephrase / remove.

Reply:
I've decided to take most of this out since it was actually duplicative of the prior section, with the exception of plan comparison, which I've edited into the above text.